### ISSUE TITLE
Replace "x Data Saved" Popup with Tray Notification

### ISSUE THREAD

**Issue by DavidBaldsiefen** [hzi-braunschweig/SORMAS-Project#4729]
### Situation Description

Currently, whenever a case/event/contact/... is saved a notification popup shows up like this:
![image](https://user-images.githubusercontent.com/4655486/110785701-876b4d80-826b-11eb-8e48-ff41e271d01d.png)

This is not very pretty and can also get in the way of the current workflow (even though it disappears after a few seconds)

### Proposed Change

Replace the popup with a less obstructive tray notification (see bottom right):

![image](https://user-images.githubusercontent.com/4655486/110785899-c39eae00-826b-11eb-8370-ec74603cd6bf.png)

The Tray notification also disappears after a few seconds

**Comment by kwa20** [comment]
@DavidBaldsiefen does this include the placement of the notification? The blue colours are definitely nicer but I'd prefer having it centred because it's otherwise easily overlooked.

**Comment by DavidBaldsiefen** [comment]
> @DavidBaldsiefen does this include the placement of the notification? The blue colours are definitely nicer but I'd prefer having it centred because it's otherwise easily overlooked.

Yes the notification would be at the bottom right, as in the screenshot. I agree it might be overlooked but then again, would a user really notice if a saved-confirmation does not appear at the moment?

**Comment by kwa20** [comment]
At the moment, saving sample data gets shown like in the second screenshot already and new staff always get a little confused about that. It might be different if we change all the notifications but they always wonder if this message is misplaced. Making this the default might change this but I can't tell for sure.

**Comment by MateStrysewske** [comment]
@kwa20 Would it mabye make sense to style the tray notification differently so that it's more apparant to the user? Right now it's REALLY subtle and very easy to miss.

**Comment by DavidBaldsiefen** [comment]
I restyled it slightly for another issue, and i think it's definitely already more visible now:

![Screenshot_2021-03-18 SORMAS](https://user-images.githubusercontent.com/4655486/111599442-8f2d7380-87d0-11eb-8142-39624abe11dc.png)

Not sure if that's enough though.

**Comment by bernardsilenou** [comment]
@DavidBaldsiefen  what about using blue filling and white text, just like the save icon? since the background is white, then this may be more visible.

**Comment by DavidBaldsiefen** [comment]
I like that Idea! 

This is how it would look:

![image](https://user-images.githubusercontent.com/4655486/111764868-f9631880-88a3-11eb-9027-1b9fd01ef7e3.png)

![image](https://user-images.githubusercontent.com/4655486/111765287-7c846e80-88a4-11eb-8bed-d769592c557a.png)
